src/main/java/com/amazon/aocagent/validators/MetricValidator.java

```diff
@@ -78,7 +78,8 @@ public void validate(Context context) throws Exception {
 
   private List<Metric> getExpectedMetricList(Context context) throws IOException {
     // get expected metrics as yaml from config
-    String yamlExpectedMetrics = mustacheHelper.render(context.getExpectedMetric().name(), context);
+    String yamlExpectedMetrics =
+        mustacheHelper.render(context.getExpectedMetric(), context);
```

Comment:
nit: I think can we hide the lib name from variable/class name in our code. sometimes it could get confused if it is a class/var for YMAL or Mustache lib. 
some suggestion like changing ```yamlExpectedMetrics``` to ```expectedMetrics```
```MustacheHelper``` to ```TemplateHelper```

Reply:
yeah, that makes sense, i will change it to TemplateHelper  and expectedMetrics